ci/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
I'd somehow include the word travis in the filename of this script - it depends on environment variables set by travis and we also support more CI environments. I guess there's an argument for this to be abstract across multiple CI environments but let's save that for another day.

You're totally right though - this script is SO much better than inlining into the yaml! I also like the clarifying comments...

(I guess if/when travis goes to ubuntu 20.04 most of the 32 bit packages won't be available any more but again one for another day)

Reply:
That sounds like a good idea to me. I have renamed this script and I have split it in two scripts - travis-build.sh and travis-install.sh.